pkg/kubectl/cmd/logs.go

```diff
@@ -223,26 +254,41 @@ func (o LogsOptions) Validate() error {
 }
 
 // RunLogs retrieves a pod log
-func (o LogsOptions) RunLogs() error {
-	switch t := o.Object.(type) {
-	case *api.PodList:
-		for _, p := range t.Items {
-			if err := o.getLogs(&p); err != nil {
-				return err
+func (o LogsOptions) RunLogs(out io.Writer, args []string) error {
+	if len(o.PrefixMatchList) > 1 {
+		firstPrint := true
+		fmt.Fprintf(out, "No pod named %s, do you mean? ", args[0])
```

Comment:
nit: s/do you mean/did you mean/
Also, what if we have more than two matches here, maybe we could suggest the closest match / all matches up to a certain amount?